anaconda.spec.in

```diff
@@ -168,6 +150,35 @@ Obsoletes: booty <= 0.107-1
 The anaconda-core package contains the program which was used to install your
 system.
 
+%package install-env-deps
```

Comment:
There is no summary: 
`BUILDSTDERR: error: Summary field must be present in package: anaconda-install-env-deps`

Reply:
Fixed, thanks! :)